### PR TITLE
2015 Java day1 part1

### PR DESCRIPTION
**Pull request by SimonPrins**
I made a simple Java solution of day1 part1 for the 2015 edition.
I intend to upload java solutions to other problems later.

### CODE REVIEW

File: 2015/java/01/part1.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			// Print the answer.
+			System.out.println(answer);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
+	
+	public static int getAnswer(String line) {
+		int result = 0;
+		// Go through the input, one characetr at a time.
```
Comment (ChrisPenner) [inline 2015/java/01/part1.java:21]: Typo :P

**Comment by ChrisPenner** [comment]
Go ahead and squash these two into a single commit and I'll send it on through :) Cheers!

We'll need java answers for 2016 as well ;)

**Comment by SimonPrins** [comment]
I believe this should fix the commits.
I am rather new to git, so it took a while to figure out how to squash them. XD
I'll do some solutions for 2016 too, but I have to rework my solutions a bit first.

**Comment by ChrisPenner** [comment]
We're all learning something or other, glad you figured it out!
Cheers!

**Comment by SimonPrins** [comment]
Another question, I want to use an additional class for my solution for day 1 of 2016. Can I put that in a seperate file, or should I keep everything in one file per part? In noticed that the existing solutions all use one file. On the other had it is a standard in Java to mostly use seperate files for each class. Which would you prefer?

**Comment by ChrisPenner** [comment]
Try to keep it all in one file, Just makes it easier for people to read :)

**Pull request merged**